_data/privacy-policy.yml

```diff
@@ -2,7 +2,7 @@
   items:
     - title:
       ans: |
-        <p>Gruntwork, Inc. (“Gruntwork,” “we,” “our” and/or “us”) respects your privacy and is committed to protecting it through our compliance with this privacy policy (the “Privacy Policy”). Gruntwork is a DevOps platform designed to enable software teams to quickly launch production-grade cloud infrastructure. This Privacy Policy describes the types of information we may collect from and about you when you visit this website (the “Website”); use the Gruntwork services, including all associated features and functionalities, websites, user interfaces, and applications (collectively, the “Services”); engage with us through our newsletters, emails, and branded social media accounts; or otherwise interact with us. It also describes our practices for using, maintaining, protecting, and disclosing that information as well as certain legal rights you may have, subject to applicable law, and how you can exercise them.</p>
+        <p>Gruntwork, Inc. (“Gruntwork,” “we,” “our” and/or “us”) respects your privacy and is committed to protecting it through our compliance with this privacy policy (the “Privacy Policy”). Gruntwork is a DevOps platform designed to enable software teams to quickly launch production-grade cloud infrastructure. This Privacy Policy describes the types of information we may collect from and about you when you visit our website located at <a href="https://gruntwork.io/">www.gruntwork.io</a> (the “Site”); use the Gruntwork services, including our cloud-based developer portal located at <a href="https://app.gruntwork.io">www.app.gruntwork.io</a> (“the Portal”) and any additional Gruntwork-branded features and functionalities, websites, user interfaces, and applications (collectively with the Portal, the “Services”); engage with us through our newsletters, emails, and branded social media accounts; or otherwise interact with us. It also describes our practices for using, maintaining, protecting, and disclosing that information as well as certain legal rights you may have, subject to applicable law, and how you can exercise them.</p>
```

Comment:
The app URL is correct in the href, but wrong in the text: no `www` before `app.gruntwork.io`.